webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -222,6 +232,13 @@ function SystemUsers(props: Props) {
                             >
                                 {info.row.original.username}
                             </div>
+                            {info.row.original.error &&
+                            <div
+                                className='error'
+                                title={info.row.original.error.message}
+                            >
+                                {info.row.original.error.message}
+                            </div>}
```

Comment:
I'm all for changing this, but I think it's something we can follow-up on, since it wasn't currently in the designs. I know that Zubair and I have some extra time but it would depend on Abhijit to get a design together and then for us to implement it. If we can do that quickly then changing it here makes sense, but I don't want to block on this. 3/5

Reply:
> in conjunction with the valuable UX insights provided by @abhijit-singh on the error management.

@M-ZubairAhmed @abhijit-singh can you point me to the UX provided for error management? I don't see it in the Figma. 

> If we can do that quickly then changing it here makes sense, but I don't want to block on this.

@devinbinnie can you share how errors would show with the approach in this PR?

Reply:
@marianunez On this branch you'll get an error message that looks like this:
![image](https://github.com/mattermost/mattermost/assets/52460000/a0ac6bde-13fd-4ad3-bca8-1e354e51cd06)

This is comparable to what we have currently in master, which looks like this (the red box is where the error is supposed to show, but apparently it's broken):
![image](https://github.com/mattermost/mattermost/assets/52460000/15664c36-88ae-49ba-aa83-b557372624a6)

I wanted to go with this for now because a) it doesn't change/remove any existing functionality and b) I believe we didn't spec out any different UX when designing this. I think the assumption might have been that the modals would cover this (which some do), but not all of the actions work the same way.